public/app/modules/home/home.controller.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Created by tremaine on 1/15/16.
+ */
+(function(){
+    'use strict';
+    angular
+        .module('home')
+        .controller('Home', Home);
+
+    Home.$inject = ['$scope', 'dataService'];
+    /**
```

Comment:
Key to good coding is documentation.  Haven't seen any so far! @tremainebuchanan